packages/app/src/components/DynamicRoot/DynamicRoot.tsx

```diff
@@ -220,7 +220,7 @@ export const DynamicRoot = ({
     >((acc, { module, importName, mountPoint, scope, config }) => {
       const Component = allPlugins[scope]?.[module]?.[importName];
       // Only add mount points that have a component
-      if (Component) {
+      if (Component || mountPoint === 'global.floatingactionbutton/component') { // Fix this
```

Comment:
One idea is that all `global.floatingactionbutton/components` needs to export a component. That could be a simple `NullComponent for know.

We should also think about the integration options for the lightspeed plugin (as a follow-up).